lib/room.js

```diff
@@ -427,6 +448,8 @@ function handleSignalingEvents(room, signaling) {
     + 'ParticipantSignalings that connect to the RoomSignaling');
   signaling.on('participantConnected', connectParticipant.bind(null, room));
 
+  signaling.on('dominantSpeakerChanged', () => room.emit('dominantSpeakerChanged', room.dominantSpeaker));
```

Comment:
Write a test case for the case where:

1. `dominantSpeaker` is `null`,
2. An invalid Dominant Speaker SID arrives, and
3. Therefore we emit "dominantSpeakerChanged", although `dominantSpeaker` is still `null`.